lagde en lite knapp for å se passord

chlar7639 opened the pull request:
trenger å fikse icons og gjøre den smooth

# :gem: Changes

Closes [Issue number]

<!-- Explain what you are changing and WHY you are changing it.
Use a list to express the changes if there are multiple:
- A change
  -  A subchange
- Another change

You should upload a picture of your changes if applicable. Drag & drop or paste from clipboard.
-->

chlar7639 committed:
lagde en lite knapp for å se passord
trenger å fikse icons og gjøre den smooth

TorgrimRL commented:
Not quite sure if you mean this is still being worked on regarding adding icon instead of a button, or that is possible future work and you think its good enough right now. 

I also suggest you look into the CONTRIBUTING.md file for branchnames and gitmojis and commit messages. 

Anyhow well done, there is no doubt this is a usefull feature and what you have implemented so far works as intended

chlar7639 committed:
lagde show password icon button, fikk ikke til å
bruke chakra-ui for å lage icons. fikk llm til å
lage custom icons. er ikke stor på color theory
så endring av farge på den må gjerne gjøres.

chlar7639 committed:
run with prettier
